src/action.ts

```diff
@@ -171,10 +171,9 @@ export async function performAction(
 	tracker: string,
 ): Promise<ActionResult> {
 	const { action, linkDir } = getRuntimeConfig();
-	const isVideo = hasVideo(searchee);
 
 	if (action === Action.SAVE) {
-		await saveTorrentFile(tracker, getTag(searchee.name, isVideo), newMeta);
+		await saveTorrentFile(tracker, getTag(searchee.name, searchee), newMeta);
```

Comment:
```suggestion
		await saveTorrentFile(tracker, getTag(searchee), newMeta);
```
redundant to pass searchee.name and searchee both, just send the searches and pull the .name out of it in getTag...

Reply:
I'm dumb. I thought about that when I first added isVideo but forgot about it here...